src/App.js

```diff
@@ -0,0 +1,57 @@
+import Headline from './components/Headline';
+import Button from './components/Button';
+
+import logo from './images/ironhack-logo-xs.png';
+import menu from './images/menu-top-xs.png';
+import icon1 from './images/icon1.png';
+import icon2 from './images/icon2.png';
+import icon3 from './images/icon3.png';
+import icon4 from './images/icon4.png';
+
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="Navbar">
+        <img id="img1" src={logo} alt="" height="50px" />
+        <img id="img2" src={menu} alt="" height="20px" />
+      </div>
+      <div className="App-header">
+        <Headline />
+
+        <p>
+          You will learn how to use the most popular frontend library, and
+          become a super Ninja developer.
+        </p>
+
+        <Button />
+      </div>
+
+      <div className="AllCards">
+        <div className="Card">
+          <img src={icon1} alt="" />
+          <h5>Declarative</h5>
+          <p>React makes ir painless to create interactive UIs.</p>
+        </div>
```

Comment:
You can move this logic to a new component named "Card", and just call that in the App.js

Reply:
I tried to do it, but it messed up with the css, so I submitted this way. 
I'm heaving trouble with components and props. But I'll catch up!